dts/bindings/cpu/riscv,b91.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+description: Telink RISC-V CPU
+
+compatible: "riscv,b91"
```

Comment:
Looking here: https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/riscv/cpus.yaml

I think it would be more appropriate to say "telink,b91" here.

You can still say `compatible = "telink,b91", "riscv";` in a DTSI file as done in the SiFive example in the above link.

Reply:
Done.

Reply:
Thanks for your quick response, but it looks like this change was done in an additional commit in the same PR. Zephyr policy is that changes in response to review should be squashed into the original commits, not put on top. Please rebase the patch series so requested changes are not handled as additional commits, but squashed appropriately.

Reply:
Done.